### ISSUE TITLE
Class on div containing bare image

### ISSUE THREAD

**Issue by davidfarmer** [rbeezer/mathbook#1158]
Bare images are allowed now.  In the HTML those become an `img` in a `div`, with the
`div` containing left-right alignment information.

Currently there is not enough vertical space between the preceding text and the image.
The best way to fix this is to put a class on the `div`, indicating that it exists as a container
to hold an image.

I'll adjust the CSS once this is in place.

**Comment by davidfarmer** [comment]
Also, the click-to-magnify is not working on those images.  I'll also fix that once
the class is added.

**Comment by rbeezer** [comment]
Excellent, this has been needed for a while.  More on the lists.

Full beta at

http://pretextbook.org/beta/2019-08-02-image-box

Section 9.9 has test examples.

Bare images go...
```
div.image-box
  img
```
Send an all-clear and I can push the changes.

Thanks,
Rob

**Comment by davidfarmer** [comment]
I had expected .image-box only around bare images
(and not those in figures), but it is fine this way
and I have made appropriate adjustments to the CSS
and Javascript.

Not sure if this merits an announcement.

Maybe let's not use .image-box for things in the banner or
other places that are not part of the text.

**Comment by rbeezer** [comment]
Thanks, David!  I had not thought too carefully about bare versus in-figure.  The change was in just one place, so that is what happened.  I'm glad it is OK.

The distinction is bare/figure versus side-by-side panel.  One-shot versus multiple.  I'm putting every image in a box and letting the box control size and position.  Complete I think for HTML, and maybe complete for LaTeX (need to check).  The  `div.panel`  of the sidebyside plays the role of the  `div.image-box`.

Yes, definitely do not use this for images that are part of the interface or overall structure - these are just author-supplied images.  In content.

About to push changes.  Glad to have click-to-magnify back.  Sorry I unwittingly broke that.

See #1160

**Comment by rbeezer** [comment]
Should have added.  I don't think this needs an announcement.  Hopefully nobody notices.  ;-)  Any problems will likely get reported quickly.

**Comment by rbeezer** [comment]
Pushed and live on website:  abdc405873c20c15f1fd5e91f89a505e5196f582
